EnglishFilter/sections/whitelist_stealth.txt

```diff
@@ -42,6 +42,17 @@
 !
 !
 !
+! https://github.com/AdguardTeam/CoreLibs/issues/1645 [Stealth Mode - Protect from DPI]
+! TODO: remove when this issue will be fixed
+!#if (adguard_app_windows || adguard_app_mac || adguard_app_android)
+@@||bcee.snet.lu^$stealth
+@@||pravo.by^$stealth
+@@||kun.uz^$stealth
+@@||mfah.org^$stealth
+@@||aztelekom.az^$stealth
+@@||gosuslugi.ru^$stealth
+@@||filecity.co.kr^$stealth
```

Comment:
In fact, it's not a complete list from the [task](https://github.com/AdguardTeam/CoreLibs/issues/1645) and comments. What about the others?